app/controllers/saml_idp_controller.rb

```diff
@@ -133,9 +133,25 @@ def capture_analytics
       matching_cert_serial: saml_request.service_provider.matching_cert&.serial&.to_s,
       requested_nameid_format: saml_request.name_id_format,
     )
+
+    if result.success? && saml_request.signed?
+      # Logging to indicate if a validation bug fix will create a potentially breaking change
+      analytics_payload[:encryption_cert_matches_matching_cert] =
+        encryption_cert_matches_matching_cert?
+    end
+
     analytics.saml_auth(**analytics_payload)
```

Comment:
can we update the documentation in `analytics_events.rb` for this new `encryption_cert_matches_matching_cert` key

Reply:
updated in https://github.com/18F/identity-idp/pull/10765/commits/09211df4d332444414170945b5441a5f190eb1ab !